src/main/scripts/jbang

```diff
@@ -61,7 +63,12 @@ case "$(uname -m)" in
   armv7l)
     arch=arm;;
   *)
-    echo "Unsupported Architecture: $(uname -m)" 1>&2; exit 1;;
+    ## AIX gives a machine ID for `uname -m` but it only supports ppc64
```

Comment:
Are you saying that on AIX `uname -m` does not return "ppc64"?

Reply:
I think I've found the answer already online: no it doesn't. It seems to be a long list of possible version numbers.

Reply:
https://sites.ualberta.ca/dept/chemeng/AIX-43/share/man/info/C/a_doc_lib/cmds/aixcmds5/uname.htm

Reply:
Correct - `uname -p` does though

Reply:
So should we be using `uname -p` or is there some other hardware where that won't work?

Reply:
NB: Git Bash on WIndows returns "unknown" for `-p`

Reply:
Yeah there isn't a sensible standard choice to use universally unfortuantely (dealt with this many times in the past - here's one example which had more options: https://github.com/sxa/qinstall/blob/26f3e7e4cc4993596da416f32ffbea1138b29820/qinstall#L34 - I did use `uname -p` on AIX there) which is why I figured just basing it on the OS name would be a reasonable solution for jbang (for now ...)

Reply:
Indeed, I had actually considered a solution where we'd just use both the value of -m and -p and if one of those matches the pattern it would be okay. Concatenating the two values would work as well indeed.

Reply:
Annoyingly `uname -p` returns `powerpc` on AIX so while I could change it to say `arch="$(uname -p)"` within the AIX section it would still have to be translated to get the correct name for the adoptopenjdk download - so we'd likely end up with `arch="$(uname -p | sed s/powerpc/ppc64/)"` in the AIX case. I can change to that if you'd prefer it over a fixed `arch=ppc64`

Reply:
No, don't worry :-)